test/modules/aggregation-layers/contour-layer/marching-squares.spec.js

```diff
@@ -390,6 +391,185 @@ const GETVERTEX_TESTS = [
     cellSize: [12, 24],
     vertices: [[[106, 236], [106, 224], [112, 212], [118, 212], [118, 224], [112, 236]]],
     type: CONTOUR_TYPE.ISO_BANDS
+  },
+  // linear interpolation flag set true
```

Comment:
Lets reduce the code duplication here, you can have each test case defined in following way :
```
{
  gridOrigin: [..]
  // all other props needed for both original and smooth cases
  vertices: ...
  smoothVertices: ...
 smoothEnabled: [true, false]
}
```
And update the for loop, so each test case is run twice, once with smoothEnabled[0] and smoothEnabled[1], that way we can also visually compare how vertices change when these flag is on/off.